tests/unit/python/foglamp/common/storage_client/test_exceptions.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test foglamp/common/storage_client/exceptions.py """
+
+import pytest
+
+from foglamp.common.storage_client.exceptions import StorageClientException
+from foglamp.common.storage_client.exceptions import *
+
+__copyright__ = "Copyright (c) 2018 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("common", "storage_client")
+class TestStorageClientExceptions:
+
+    def test_init_StorageClientException(self):
+
+        with pytest.raises(Exception) as excinfo:
+            raise StorageClientException()
+        assert excinfo.type is TypeError
+        assert "__init__() missing 1 required positional argument: 'code'" == str(excinfo.value)
+
+    def test_default_init_StorageClientException(self):
+        with pytest.raises(Exception) as excinfo:
+            raise StorageClientException(40)
+        assert excinfo.type is StorageClientException
+        assert issubclass(excinfo.type, Exception)
+
+        try:
+            raise StorageClientException(40)
+        except Exception as ex:
+            assert ex.__class__ is StorageClientException
+            assert issubclass(ex.__class__, Exception)
+            assert 40 == ex.code
+            assert ex.message is None
+
+    def test_init_args_StorageClientException(self):
+        with pytest.raises(Exception) as excinfo:
+            raise StorageClientException(code=11, message="foo")
+        assert excinfo.type is StorageClientException
+        assert issubclass(excinfo.type, Exception)
```

Comment:
Same assertions called below? If that is just for demo purpose for pytest raise wrapper, can we comment it out and have single place of all assertions?

Reply:
> # pytest raises wrapper allow only type, value and traceback info

So yes, we can say its for that explanation. I don;t think we should remove these or comment